modelci/hub/manager.py

```diff
@@ -246,7 +71,8 @@ def _get_remote_model_weights(models: List[ModelBO]):
     # group by (task, architecture, framework, engine) pair
     pairs = set(map(lambda x: (x.task, x.architecture, x.framework, x.engine), models))
     model_groups = [
-        [model for model in models if (model.task, model.architecture, model.framework, model.engine) == pair] for pair in pairs
+        [model for model in models if (model.task, model.architecture, model.framework, model.engine) == pair] for pair
+        in pairs
```

Comment:
Is there a syntax error? By the way, you should not break a long line in this way.
```python
[
    [
        model for model in models if (model.task, model.architecture, model.framework, model.engine) == pair
    ] for pair in pairs
]
```

Reply:
Or just use a for loop